cookbooks/bcpc/recipes/ufw.rb

```diff
@@ -1,78 +0,0 @@
-#
```

Comment:
This is needed by the VM cluster so that the bootstrap can act as the fake router for VMs.